aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -135,11 +135,93 @@ public class LogBufferDescriptor
      */
     public static final int LOG_DEFAULT_FRAME_HEADER_OFFSET;
 
+
     /**
      * Maximum length of a frame header.
      */
     public static final int LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH = CACHE_LINE_LENGTH * 2;
 
+
+    /**
+     * Offset within the log metadata where the term offset is stored.
+     */
+    public static final int LOG_TERM_OFFSET_OFFSET;
```

Comment:
A natural place to store this information would the default frame header, i.e.:
```java
        defaultDataHeader.sessionId(sessionId).streamId(streamId).termId(initialTermId).termOffset(termOffset);
        storeDefaultFrameHeader(logMetaData, defaultDataHeader);
```

Reply:
Ok, will add it to the frame header.